### DISCUSSION TITLE
Pro Plan:  Vercel fails to accept 300s maxDuration argument in verecl.json (terminal says maxDuration must be between 1-10)

### DISCUSSION THREAD

**Discussion by zadamg** [anuraghazra/github-readme-stats#3305] [Q&A] [answered]
### Describe the bug

Vercel fails to accept maxduration argument of 300 in verecl.json on sveltekit.

```
C:\...\twilitype>vercel
Vercel CLI 28.20.0
Error: The value for maxDuration must be between 1 second and 10 seconds, in order to increase this limit upgrade your plan: https://vercel.com/pricing
```

But, I have already upgraded to the pro plan.  This is my vercel.json:

```
{
    "functions": {
      "api/test.js": {
        "memory": 3008,
        "maxDuration": 300 
      },
      "api/*.js": {
        "memory": 3008,
        "maxDuration":300 
      }
    }
  }
  ```
  
  

### Expected behavior

According to [maxDuration docs](https://vercel.com/docs/functions/serverless-functions/runtimes#max-duration), upgrading to Pro plan, one expects to be able to set the maxDuration to 300 seconds without problems/errors.

There is no copy referencing a wait time, or delay after upgrading, or any additional information.  Therefore, the expectation is instantaneous access to maxDuration 300s at the moment of CC approval.

**Comment by qwerty541** [comment, marked as answer]
Hey, @zadamg! You should contact Vercel support https://vercel.com/contact about this problem. It it not related to this repository.

**Reply by zadamg** [reply]
Yeah, I totally submitted this on the wrong page haha.  Oopsies......
